Notes-Azure.adoc

```diff
@@ -0,0 +1,147 @@
+
+$ az vm list -g "scf-resource-group" | jq '.[] | select (.tags.poolName | contains("agent")) | .name' | \
+  xargs -i{} az vm run-command invoke \
+    --resource-group "scf-resource-group" \
+    --command-id RunShellScript \
+    --scripts "sudo update-grub" --name {}
+
+$ az vm list -g "scf-resource-group" | jq '.[] | select (.tags.poolName | contains("agent")) | .name' | \
+  xargs -i{} az vm restart --no-wait \
+    --resource-group "scf-resource-group" \
+    --name {}
+----
+
+* Create a new public ip address and write it down.
++
+[source,bash]
+----
+$ az network public-ip create --resource-group scf-resource-group --name scf-public-ip --allocation-method Static
+----
+
+
+* Extract the name of one of the kubes agent nic and write it down
++
+[source,bash]
+----
+$ az network nic list --resource-group scf-resource-group | grep name | grep agent | grep 0
+----
+
+* Attach the public ip to the master kubes node by replacing the term _<nic-name>_ by the one noted down before and write down the value of _privateIpAddress_.
++
+[source,bash]
+----
+$ az network nic ip-config update --resource-group scf-resource-group --nic-name <nic-name> --name ipconfig1 --public-ip-address scf-public-ip
+----
+
+
+* Extract the security group name of the master network security group and write it down
++
+[source,bash]
+----
+$ az network nsg list --resource-group=scf-resource-group | jq -r '.[] | .name' | grep master
+----
+
+
+* Create required security groups and replace _<security-group>_ by the name wrote down in the step before.
++
+[source,bash]
+----
+$ export NSG_NAME=<security-group>
+$ az network nsg rule create --resource-group scf-resource-group --priority 200 --nsg-name $NSG_NAME --name scf-80 --direction Inbound --destination-port-ranges 80 --access Allow
+$ az network nsg rule create --resource-group scf-resource-group --priority 201 --nsg-name $NSG_NAME --name scf-443 --direction Inbound --destination-port-ranges 443 --access Allow
+$ az network nsg rule create --resource-group scf-resource-group --priority 202 --nsg-name $NSG_NAME --name scf-4443 --direction Inbound --destination-port-ranges 4443 --access Allow
+$ az network nsg rule create --resource-group scf-resource-group --priority 203 --nsg-name $NSG_NAME --name scf-2222 --direction Inbound --destination-port-ranges 2222 --access Allow
+$ az network nsg rule create --resource-group scf-resource-group --priority 204 --nsg-name $NSG_NAME --name scf-2793 --direction Inbound --destination-port-ranges 2793 --access Allow
+----
+
+=== Deploying SUSE Cloud Application Platform
+
+Once the Kubernetes environment is prepared proceed with deployment of SUSE Cloud Application Platform as described in the https://www.suse.com/documentation/cloud-application-platform-1/book_cap_deployment/data/book_cap_deployment.html[Deployment Guide].
```

Comment:
It would be good to add some specific information what values to use in the configuration of the deployment and make it explicit where to get them from based on the instructions from this document.